### ISSUE TITLE
[Bug] tooltip with colon are empty

### ISSUE THREAD

**Issue by ikabod** [uikit/uikit#3390]
### UIkit version

3.0.0-rc.2

### Reproduction Link

https://codepen.io/anon/pen/NMJBoz

### Steps to reproduce
Hover over the both buttons

### What is Expected?
Tooltip should always appear

### What is actually happening?
Tooltip with colon is empty

**Comment by MartinKravec** [comment]
WTF, open ti and fix it!!!

**Comment by aarongerig** [comment]
@MartinKravec I don't think this is fixable. Since any option keys in attributes are followed by a colon, UIkit would look after an option named "Hello world", which doesn't exists, so it doesn't do anything. Like I said, you can use the proper option key `title` and prepend it to your text. That solves this issue. Here's a codepen: https://codepen.io/aarongerig/pen/jgrjXL
